microbench/src/main/java/io/netty/microbench/http2/NoopHttp2RemoteFlowController.java

```diff
@@ -84,4 +84,8 @@ public ChannelHandlerContext channelHandlerContext() {
     @Override
     public void channelWritabilityChanged() throws Http2Exception {
     }
+
+    @Override
+    public void updateDependencyTree(int childStreamId, int parentStreamId, short weight, boolean exclusive) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1186)